kitchen-tests/kitchen.yml

```diff
@@ -21,7 +21,7 @@ lifecycle:
     - remote: /opt/chef/embedded/bin/ohai -v
     - remote: /opt/chef/embedded/bin/gem install appbundler appbundle-updater
     - remote: /opt/chef/embedded/bin/appbundle-updater chef ohai <%= File.readlines('../Gemfile.lock', File.expand_path(File.dirname(__FILE__))).find { |l| l =~ /^\s+ohai \((\d+\.\d+\.\d+)\)/ }; 'v' + $1 %> --tarball --github chef/ohai
-    - remote: /opt/chef/embedded/bin/appbundle-updater chef chef <%= ENV['TRAVIS_COMMIT']  || %x(git rev-parse HEAD).chomp %> --tarball --github chef/chef
```

Comment:
I assume that travis is now falling back to the git method now right?

Reply:
yes